packages/frontend/src/services/tokenExchange/FungibleTokenExchange.js

```diff
@@ -0,0 +1,107 @@
+import { NEAR_TOKEN_ID } from '../../config';
+import { wallet } from '../../utils/wallet';
+import { fungibleTokensService } from '../FungibleTokens';
+import refFinanceContract from './RefFinanceContract';
+import { NEAR_ID, isNearSwap, replaceNearIfNecessary } from './utils';
+
+class FungibleTokenExchange {
+    #exchange;
```

Comment:
What exactly mean `#`? I never see it used in javascript code.
Like a **private**?

Reply:
Use `this._value` or `this._method`, it is classic unspoken standard in js and another programming languages for private methods